reactioncommerce/reaction
# Polish currency formatting

Resolves #1868 

### To test
1. Run `reaction`
1. Enable PLN currency unit from i18n menu option.
1. Select PLN as the default currency unit from the same menu.
1. Go to `example-product` and see the correct format.
